### PR TITLE
6927: updated button for message list

### PR DESCRIPTION
**Pull request by nikola-gladovic**
Signed-off-by: Nikola <nikola.gladovic@nextcloud.com>

#6927

### CODE REVIEW

File: src/components/MessagesList/MessagesList.vue [hunk 1]
```diff
@@ -49,14 +49,15 @@ get the messagesList array and loop through the list to generate the messages.
 				:count="15" />
 		</template>
 		<transition name="fade">
-			<button v-show="!isChatScrolledToBottom"
+			<Button v-show="!isChatScrolledToBottom"
+				type="secondary"
 				:aria-label="scrollToBottomAriaLabel"
 				class="scroll-to-bottom"
```
Comment (nickvergessen) [inline src/components/MessagesList/MessagesList.vue:55]: You can remove these rules from the style definition:
```
width: 44px;
height: 44px;
```
```
	padding: 0;
	margin: 0;
	display: flex;
	align-items: center;
	justify-content: center;
```

File: src/components/MessagesList/MessagesList.vue [hunk 2]
```diff
@@ -49,14 +49,15 @@ get the messagesList array and loop through the list to generate the messages.
...
 				@click="smoothScrollToBottom">
 				<ChevronDown decorative
 					title=""
 					:size="20" />
```
Comment (nickvergessen) [inline src/components/MessagesList/MessagesList.vue:59]:
```suggestion
				<template #icon>
					<ChevronDown decorative
						title=""
						:size="20" />
				</template>
```

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: src/components/MessagesList/MessagesList.vue [hunk 3]
```diff
@@ -49,14 +49,17 @@ get the messagesList array and loop through the list to generate the messages.
 				:count="15" />
 		</template>
 		<transition name="fade">
-			<button v-show="!isChatScrolledToBottom"
+			<Button v-show="!isChatScrolledToBottom"
+				type="secondary"
```
Comment (nickvergessen) [inline src/components/MessagesList/MessagesList.vue:53]: Secondary is broken and might not be fixable for 24.

So let's wait with this PR

**Review by nickvergessen — CHANGES_REQUESTED** [review]

File: src/components/MessagesList/MessagesList.vue (hunk 3 above)
Comment (nikola-gladovic) [reply src/components/MessagesList/MessagesList.vue:53]: Should we wait another PR ?

Comment (nikola-gladovic) [reply src/components/MessagesList/MessagesList.vue:53]: @marcoambrosini 
Could you please help me for this PR? ;)

Comment (marcoambrosini) [reply src/components/MessagesList/MessagesList.vue:53]: With the fix from Joas in the vue lib this should be mergeable already! I will release soon!

**Review by marcoambrosini — APPROVED** [review]

File: src/components/MessagesList/MessagesList.vue (hunk 3 above)
Comment (nikola-gladovic) [reply src/components/MessagesList/MessagesList.vue:53]: Ok, cool

Comment (nikola-gladovic) [reply src/components/MessagesList/MessagesList.vue:53]: Hi @nickvergessen 
I think it is ready to be reviewed now :)

Comment (nickvergessen) [reply src/components/MessagesList/MessagesList.vue:53]: then change the labels :)

**Pull request merged**